apps/site/src/app/(main)/schedule/components/EventRegular.tsx

```diff
@@ -93,13 +91,15 @@ export default function EventRegular({
 				</h3>
 				<EventTypeComponent eventType={eventType} />
 			</div>
+			{(organization !== undefined || hosts) && (
+				<p className="mb-2">
+					Hosted by:{" "}
+					{organization === undefined ? hosts?.join(", ") : organization}
+				</p>
+			)}
 			<p className="mb-2">
-				Hosted by:{" "}
-				{organization === undefined ? hosts?.join(", ") : organization}
-			</p>
-			<p className="mb-2">
-				{dateTimeFormat.formatRange(startTimeInPST, endTimeInPST)} PST |{" "}
-				<a href={virtual}>Meeting Link</a>
+				{dateTimeFormat.formatRange(startTimeInPST, endTimeInPST)} PST{" "}
+				{virtual && <a href={virtual}>| Meeting Link</a>}
```

Comment:
After talking to log, we won't be putting Zoom or virtual links on the schedule page, so you can remove them instead. Sorry for this inconvenience

Reply:
Oh wait we are using the same Zoom link for all.

Reply:
Oh so will this check be necessary or will we just have a hard-coded link

Reply:
this check will be necessary